packages/testing/src/execution_testing/specs/benchmark.py

```diff
@@ -104,6 +104,8 @@ def generate_repeated_code(
         # TODO: Unify the PUSH0 and PUSH1 usage.
         code = setup + Op.JUMPDEST + repeated_code * max_iterations + cleanup
         code += Op.JUMP(len(setup)) if len(setup) > 0 else Op.PUSH0 + Op.JUMP
+        # Pad the code to the maximum code size.
+        code += Op.STOP * (max_code_size - len(code))
```

Comment:
No change needed right now, but if this ever becomes an issue we could add a `pad_code` field to this class for a test to be able to disable padding.